README.md

```diff
@@ -431,7 +431,7 @@ assert_eq!(
 
 ```rust
 let query = Query::select()
-    .expr(Func::sum(Expr::tbl(Char::Table, Char::SizeH)))
+    .expr(Func::sum(Expr::col((Char::Table, Char::SizeH))))
```

Comment:
This should go directly into the master branch. Let me update it now. https://github.com/SeaQL/sea-query/commit/34169edd36447c25e2a624d6cd745da3631987ca